bot/exts/evergreen/trivia_quiz.py

```diff
@@ -226,18 +475,20 @@ async def leaderboard(self, ctx: commands.Context) -> None:
 
     @staticmethod
     async def send_score(channel: discord.TextChannel, player_data: dict) -> None:
-        """A function which sends the score."""
+        """Send the score."""
```

Comment:
will be resolved in the upcoming commit

Reply:
Actually, I don't think this function needs any explanation. But because it's required in the linting, you have to provide a dummy string.
If you mark the function as private (with a leading underscore), is the docstring not required?

Reply:
i'm not sure